README.md

```diff
@@ -26,15 +26,10 @@ For the complete feature-set please refer to our [documentation](https://nightli
 
 ### Current API version: `v1beta1`
 
-Please check out our docs to read about the [upgrade process](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/upgrade/) and our [backward compatibility guarantees](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/compatibility/).
-
-## Version Matrix
+To download the latest stable version please visit the [Flink Downloads Page](https://flink.apache.org/downloads.html).
+The official operator images are also available on [Dockerhub](https://hub.docker.com/r/apache/flink-kubernetes-operator/tags).
 
-| Operator Version     | API Version | Flink Compatibility | Operator Image                           |
-|----------------------|-------------|---------------------|------------------------------------------|
-| `latest` (main HEAD) | `v1beta1`   | 1.13, 1.14, 1.15    |                                          |
-| `release-1.0.1`      | `v1beta1`   | 1.13, 1.14, 1.15    | `apache/flink-kubernetes-operator:1.0.1` |
-| `release-0.1.0`      | `v1alpha1`  | 1.14                | `apache/flink-kubernetes-operator:0.1.0` |
+Please check out our docs to read about the [upgrade process](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/upgrade/) and our [backward compatibility guarantees](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-main/docs/operations/compatibility/).
```

Comment:
If we remove this version matrix, where the users could find the compatible Flink versions with specific operator release?

Reply:
The downloads page already shows this information. The README is only a pain in the butt during the realease process as you have to update it in addition to updating everything else:

<img width="693" alt="image" src="https://user-images.githubusercontent.com/5880972/180735886-05f02343-c4a9-4d20-97be-ecb96d2256af.png">

Reply:
Make sense.